client/signup/jetpack-connect/authorize-form.jsx

```diff
@@ -425,6 +441,11 @@ const JetpackConnectAuthorizeForm = React.createClass( {
 		);
 	},
 	render() {
+		const { queryObject } = this.props.jetpackConnectAuthorize;
```

Comment:
@johnHackworth do you think this is appropriate for checking the lack of query parameters ?